op-node/node/node.go

```diff
@@ -594,7 +602,12 @@ func initL2(ctx context.Context, cfg *config.Config, node *OpNode) (*sources.Eng
 		return nil, nil, nil, nil, fmt.Errorf("cfg.Rollup.ChainOpConfig is nil. Please see https://github.com/ethereum-optimism/optimism/releases/tag/op-node/v1.11.0: %w", err)
 	}
 
-	l2Driver := driver.NewDriver(node.eventSys, node.eventDrain, &cfg.Driver, &cfg.Rollup, cfg.L1ChainConfig, cfg.DependencySet, l2Source, node.l1Source,
+	var followUpstreamSource driver.FollowUpstreamSource
+	if node.cfg.Sync.UnsafeOnly {
+		followUpstreamSource = driver.NewL2ELFollowSource(node.l2FollowSource, node.l1Source)
```

Comment:
Same comment as earlier - we should be able to follow the L1 source through the L2 Follow Source.